### PR TITLE
Vendor update

### PR DESCRIPTION
**Pull request by tuommaki**
Update vendor to incorporate latest updates including operatorkit context
memory leak fix.

### CODE REVIEW

**Comment by MarcelMue** [comment]
More build errors :/

File: integration/setup/provider.go [hunk 1]
```diff
@@ -22,12 +22,6 @@ func provider(ctx context.Context, config Config) error {
 			},
 			Secret: chartvalues.AzureOperatorConfigSecret{
 				AzureOperator: chartvalues.AzureOperatorConfigSecretAzureOperator{
-					CredentialDefault: chartvalues.AzureOperatorConfigSecretAzureOperatorCredentialDefault{
```
Comment (tuommaki) [inline integration/setup/provider.go]: Is this fine to be just deleted? That field doesn't exist in upstream library any more and tests pass with this.

Comment (MarcelMue) [reply integration/setup/provider.go]: It got deleted but I am not sure if something needs to be added to the e2e test then. Pawel worked on that for AWS.

Comment (MarcelMue) [reply integration/setup/provider.go]: See his changes here:
https://github.com/giantswarm/aws-operator/pull/1880/files

Comment (tuommaki) [reply integration/setup/provider.go]: Ok. This got fixed here: https://github.com/giantswarm/azure-operator/pull/559/files

**Comment by tuommaki** [comment]
CI Passes build. E2e expected to fail so far. Going with this.

**Pull request merged**